chrome_manifest_v3/model.js

```
@@ -245,16 +245,18 @@ Gmail2Trello.Model.prototype.loadTrelloMembers = function (boardId) {
   this.trello.members = null;
 
   Trello.get(
-    'boards/' + boardId + '/members',
+    `boards/${boardId}/members`,
     { fields: 'id,fullName,username,initials,avatarUrl' },
     function (data) {
       let me = self.trello.user;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'me' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)